dts/arm/acsip/doc/s76s.rst

```diff
@@ -0,0 +1,251 @@
+.. _acsip_s76s:
+
```

Comment:
This is probalby better kept in the boards dir for now.

Reply:
OK, I will move it over to the boards directory.  It just kind of seemed appropriate for the SiP / SoC info to be connected to the SiP / SoC definition.